src/Components/ChangePassword/ChangePassword.tsx

```diff
@@ -55,87 +50,78 @@ export default function ChangePass({ handleClose }:any) {
+                {...register("oldPassword", {
+                  required: true,
+                })}
+              />
+              <hr className="text-white" />
 
-                {errors.oldPassword &&
-                  errors.oldPassword.type === "required" && (
-                    <span className="text-danger">oldPassword is required</span>
-                  )}
-              </div>
-              <div className="form-group my-1 position-relative">
-                <p className="color my-0" style={{ fontSize: '14px' }}>New Password</p>
+              {errors.oldPassword && errors.oldPassword.type === "required" && (
+                <span className="text-danger">oldPassword is required</span>
+              )}
+            </div>
+            <div className="form-group my-1 position-relative">
+              <p className="color my-0" style={{ fontSize: "14px" }}>
+                New Password
+              </p>
 
-                <input
-                  placeholder="New Password"
-                  className="form-control ps-4 mb-1"
-                  type="password"
-                  {...register("newPassword", {
-                    required: true,
-                  })}
-                />
-                 <hr className="text-white" />
+              <input
+                placeholder="New Password"
+                className="form-control ps-4 mb-1"
+                type="password"
+                {...register("newPassword", {
+                  required: true,
+                })}
+              />
+              <hr className="text-white" />
 
-                {errors.newPassword &&
-                  errors.newPassword.type === "required" && (
-                    <span className="text-danger">newPassword is required</span>
-                  )}
-              </div>
-              <div className="form-group my-1 position-relative">
-                <p className="color my-0" style={{ fontSize: '14px' }}>Confirm Password</p>
-                <input
-                  placeholder="Confirm New Password"
-                  className="form-control ps-4 mb-1"
-                  type="password"
-                  {...register("confirmNewPassword", {
-                    required: true,
-                  })}
-                />
-                   <hr className="text-white" />
+              {errors.newPassword && errors.newPassword.type === "required" && (
+                <span className="text-danger">newPassword is required</span>
+              )}
+            </div>
+            <div className="form-group my-1 position-relative">
+              <p className="color my-0" style={{ fontSize: "14px" }}>
+                Confirm Password
+              </p>
+              <input
+                placeholder="Confirm New Password"
+                className="form-control ps-4 mb-1"
+                type="password"
+                {...register("confirmNewPassword", {
+                  required: true,
```

Comment:
you need to validate that the confirmNewPassword matches the new password using the validate method here